<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loading</class>
 <widget class="QWidget" name="Loading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loading</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1. 预个人化</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2. 后个人化</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3. 检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="prepersonal_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>preperson</string>
   </property>
  </widget>
  <widget class="QLabel" name="postpersonal_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>postperson</string>
   </property>
  </widget>
  <widget class="QLabel" name="check_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
